tests/Unit/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Test_Tov.cpp

```diff
@@ -190,6 +247,9 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.AnalyticSolutions.Gr.Tov",
   }
 
   test_tov_dp_dr(*equation_of_state);
+  test_baumgarte_shapiro();
```

Comment:
I don't see the point in reproducing figures to a part in 10^4.  Just verify the solutions satisfy Einstein's equations  (see equations in sec 1.4.3 in Shibata's NR book)

Reply:
I found it reassuring when I ran this test against values printed in the Baumgarte/Shapiro book 🤷‍♂️ I don't have a strong opinion here but I don't see a reason to delete this test. I agree the Einstein constraints should be tested of course (next PR).